01-Git_Bash_Make/script

```diff
@@ -0,0 +1,27 @@
+#! /bin/sh
+
+#Create folders tmp/guess_a_number/
+mkdir -p tmp/guess_a_number
```

Comment:
please, define /tmp/guess_a_number as variable

Reply:
Fixed it.